middle_end/debuginfo.ml

```diff
@@ -95,6 +95,9 @@ let compare dbg1 dbg2 =
   in
   loop (List.rev dbg1) (List.rev dbg2)
 
+let hash t =
```

Comment:
Wouldn't it work to use Hashtbl.hash_param with sufficiently high numbers?  Or are the hard-coded caps too low?